source/administration/compliance-export.rst

```diff
@@ -0,0 +1,51 @@
+Compliance Export Beta (E20 Add-On)
+=====================================
+
+Available as an `add-on to Enterprise Edition E20 <https://about.mattermost.com/pricing/>`_.
+
+This feature enables compliance exports to be produced from the System Console, containing all messages including those made in direct message channels, as well as file uploads.
+
+By default, Mattermost stores all message history providing an unlimited search history to admins and end users. In Enterprise Edition E20, you may set a `custom data retention policy <https://docs.mattermost.com/administration/data-retention.html>`_ for how long messages and file uploads are kept in Mattermost channels and direct messages.
+
+Those Enterprise deployments who want to archive history beyond the data retention period can enable this add-on to export compliance reports to third-party systems. Integration with Actiance Vantage is currently supported, with integrations with other systems such as GlobalRelay in the roadmap.
+
+.. note::
+  This feature will replace the existing :doc:`Compliance feature <compliance>` in a future release. Compliance exports to CSV will continue to be available in Enterprise Edition E20.
+
+.. toctree::
+    :maxdepth: 2
+
+Set Up Guide
+--------------
+
+1. Go to **System Console > Advanced > Compliance Export (Beta)**.
+2. Enable compliance exports, then set the start time of the daily scheduled compliance export job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.
+3. Set the export file format. Currently, export format to Actiance XML is supported, but support for the GlobalRelay EML format and the Mattermost CSV format is scheduled for a future release.
+4. Set the export directory to write compliance export files to. Must be a subdirectory of the Local Storage Directory. Mattermost must have write access to this directory, and the path that you set must not include a filename.
+5. Save the settings. You’re now all set!
+
+The compliance exports do not contain messages sent before the feature was enabled, but you can export past history via the ``export`` :doc:`command line tool <command-line-tools>`.
```

Comment:
Consider adding a note that if you export history from before the feature was enabled, the reports will be less accurate: more users will be listed as having been in a room than were actually in that room at export time.

Reply:
> from before the feature was enabled

Is it from before v4.5 is released or from before the feature is enabled? Since `ChannelMemberHistory` is now logged for everyone?